src/simd.jl

```diff
@@ -36,15 +36,15 @@ const SIMD_CHUNKS = (1, 2, 3, 4, 5, 6, 8, 9, 10, 12, 16, 17, 18, 20, 24, 32, 33,
 # factors for the symmetric tensors, return a quote
 function symmetric_factors(order, dim, T)
     if order == 2
-        dim == 1 && return :(SVec{1, T}((T(1),)))
-        dim == 2 && return :(SVec{3, T}((T(1),T(2),T(1))))
-        dim == 3 && return :(SVec{6, T}((T(1),T(2),T(2),T(1),T(2),T(1))))
+        dim == 1 && return :(SVec{1, $T}(($T(1),)))
```

Comment:
We were just lucky we called the type parameter `T` in the functions that used this :p